tests/parse/test_parse_clnsig.py

```diff
@@ -227,3 +228,42 @@ def test_is_pathogenic_no_annotation(cyvcf2_variant):
 
     ## THEN assert that The variant should be loaded
     assert pathogenic == False
+
+
+def test_parse_clinsig_vep97(
```

Comment:
Perhaps some kind of test with both INFO and CSQ entries as well, since a lot of the logic above is focused on working around that?

Reply:
I could add a variant with a complicated clinvar situation to the test file and check that is parsed into something we expect!

Reply:
I've introduced the check that variant with vep97 is loaded and contains the right clinvar fields along with REVEL and the rest